### ISSUE TITLE
'fallthrough' option does not work for 416 Range Not Satisfiable

### ISSUE THREAD

**Issue by bminer** [expressjs/serve-static#75]
The documentation for the `fallthrough` option reads: Let client errors fall-through as unhandled requests, otherwise forward a client error.

Unfortunately, once a SendStream emits a 'file' event, this behavior is disabled.  This can be problematic because the 416 Range Not Satisfiable error does not "fallthrough".  Why is this treated differently than a 404?

Also, I'm not sure if this is a bug with send dependency or with serve-static itself.  Please advise.  Thanks!

**Comment by dougwilson** [comment]
The current behavior is correct. I guess the bug is bad documentation. I will try to follow up more later when I'm at a computer, but wanted to get you a quick answer.

**Comment by bminer** [comment]
Never mind... I think I misunderstood the `fallthrough` option when I wrote this issue.  I now presume that the purpose is simply to allow other middleware to handle the request if a file cannot be served by the static middleware (i.e. because it doesn't exist).  If a Range header is invalid, that's more of an Error that should be passed to `next(err)`... not something that another middleware should deal with.

Closing this issue.  Sorry about the confusion.
